examples/acados_template/python/soft_constraints/generate_c_code.py

```diff
@@ -235,8 +235,8 @@ def export_terminal_nonlinear_constraint():
 ocp.solver_config.nlp_solver_type = 'SQP'
 
 # set header path
-ocp.acados_include_path  = '/usr/local/include'
-ocp.acados_lib_path      = '/usr/local/lib'
+ocp.acados_include_path  = '~/acados/include'
```

Comment:
I guess it would be great to initialize the paths in the constructor, i.e. here:
https://github.com/acados/acados/blob/e5865e45d2d4b2c51fc735ab2b55b6efe4b8a19d/interfaces/acados_template/acados_template/acados_ocp_nlp.py#L1281
Is it possible to get the `ACADOS_ROOT_DIR` there?

Reply:
not so easy, I think. The Python package can get the path to its install location, but not the path to  the source code, which would then provide the location to default install location. An alternative would be to have a local script that sets an environment variable or to have the same variable set on a system-wide basis, but I still prefer the current approach to both of these alternatives. @tmmsartor maybe has some (additional) suggestions here? :)